tests/microbenchmarks.py

```diff
@@ -4,6 +4,7 @@
 
 from utils import *
 
+
 def test_small_integers(test_case):
     print("Running", "test_small_integers")
     test_case.runAsJava(adjust("""
```

Comment:
At column 25: (F405) 'adjust' may be undefined, or defined from star imports: utils